packages/spec/src/spec.emu.html

```diff
@@ -84,6 +85,14 @@ <h1>Lexical Grammar</h1>
     `_`
     > any assigned Unicode code point other than U+FFFD greater than U+007F that does not have any of the following property values: General_Category=Surrogate, Control or Private_Use, Noncharacter_Code_Point=True, or Pattern_White_Space=True
 
+BacktickedIdentifierChars:
+    BacktickedIdentifierChar BacktickedIdentifierChars?
+
+BacktickedIdentifierChar :
+    SourceCharacter but not one of ``` or `\` or LineTerminator
+    `\` EscapeCharacter
```

Comment:
My question above amounts to: do we need this production?

Reply:
As discussed, if we want parity between string and backticked identifier, we will need allow escaping characters.